server/routes/juror-management/juror-record/juror-record.controller.js

```diff
@@ -348,58 +348,104 @@
           req.session.authentication.locCode,
```

Comment:
would this not be the problem here? I'm wondering that if trying to access a satellite record, with this location code, would fail.... 🤔

Reply:
You could be right, let me take a look into this just now, see what i can do 🤔

Reply:
you were absolutely right, have pushed a change to use session locCode

Reply:
🤓